_posts/2023-04-29-release.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Release 2023.17
+author: Priya
+date: 2023-04-29
+category: Jekyll
+layout: post
+---
+
+### Enhancements
+
+#### Email Notification for Pipeline Errors.(an alpha feature on demand)
+* Mammoth have added an email notification feature to alert users when a task or action fails during pipeline execution.
+
+
+### Defect fixes
+
+* Fixed an issue where zip file uploads were not working.
+* Fixed an issue where users were unable to move a DS/Folder to the Data Library.
+* Fixed an issue where column renames in foreign view did not get reflected in the join rule's pipeline description until the page was refreshed.
```

Comment:
Fixed an issue where column rename in foreign view did not reflect in the current view.